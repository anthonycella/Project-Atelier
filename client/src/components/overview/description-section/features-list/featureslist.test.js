import renderer from 'react-test-renderer';
import FeaturesList from './featureslist.jsx';

it('List of Features renders to the page', () => {
  const colorScheme = { textColorForeground: 'black' };
  const features = [
    { entry: 'come', value: 'on' },
    { entry: 'come', value: 'on' },
    { entry: 'come', value: 'on' },
    { entry: 'come', value: 'on' },
  ];
  const component = renderer.create(
    <FeaturesList colorScheme={colorScheme} features={features} />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
