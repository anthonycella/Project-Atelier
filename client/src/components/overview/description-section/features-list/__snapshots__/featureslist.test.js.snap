// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`List of Features renders to the page 1`] = `
<div
  className="sc-gsnTZi eocKDQ"
/>
`;
